src/brasil/gov/portal/browser/viewlets/templates/collective.nitf.image.pt

```diff
@@ -0,0 +1,20 @@
+<tal:comment tal:replace="nothing">
```

Comment:
Essa decisão foi complicada mesmo. `text_only_view` herda de [`DefaultView`](https://github.com/collective/collective.nitf/blob/ba126d6bc325e6a988422bbcfdbf654b569617f5/src/collective/nitf/browser/__init__.py#L58) ao invés da `View` padrão de collective.nitf, e precisamos do método `get_images` para ser usado em `nitf_custom_view` e `nitf_custom_displaymacros`. Achamos que assim seria melhor do que ficar duplicando código, além do que dessa forma também se no futuro for remover as customizações já fica associada à `View` padrão.